packaging/dependencies.nix

```diff
@@ -11,11 +11,28 @@
   versionSuffix,
 }:
 
+let
+  prevStdenv = stdenv;
+in
```

Comment:
Mostly for my own curiosity -- why is this bound in its own let-in scope, rather than in the one it's used in?

(If it's load bearing, it might be good to add a comment: if I were doing any kind of cleanups here at any point, I would probably want to combine the two into the same let-in scope)

Reply:
It is to avoid `stdenv = ....stdenv...`, since we only have let-rec not let.